### ISSUE TITLE
Rancher-compose hangs trying to create a container with sidekick and volumes_from to the same container

### ISSUE THREAD

**Issue by EliSnow** [rancher/rancher#1566]
Rancher: v0.25.0
Cattle: v0.58.0
User Interface: v0.29.0
Rancher Compose: v0.1.3

Here's a sample `docker-compose.yml`:

```yaml
test:
  tty: true
  image: ubuntu:14.04.2
  stdin_open: true
  volumes_from:
  - test-data
  labels:
    io.rancher.sidekicks: test-data
test-data:
  tty: true
  command:
  - cat
  image: ubuntu:14.04.2
  stdin_open: true
```

Rancher-compose freezes on this file. If the `io.rancher.sidekicks` label is moved to the `test-data` container pointing it to `test`, the service will be created. Moving the `io.rancher.sidekicks` label to the `test` container is undesirable because the resultant service will be known as `test-data` and a third container cannot link with `test`.

**Comment by deniseschannon** [comment]
@EliSnow Can you try the latest version of Rancher and Rancher-compose? You will need to upgrade both in order to use the latest rancher-compose. We just released a v0.2.1 of Rancher-compose and these files work in the latest version. 

It definitely doesn't work in v0.1.3, which is why it hadn't been documented yet. :) I'm going to add documentation about it today.

**Comment by EliSnow** [comment]
That seemed to do the trick for me. Thanks.
